qCC/ccGraphicalTransformationTool.cpp

```diff
@@ -36,17 +36,34 @@ ccGraphicalTransformationTool::ccGraphicalTransformationTool(QWidget* parent)
 	, m_toTransform("transformed")
 {
 	setupUi(this);
-
+	
+	rotComboBox->clear();
+	rotComboBox->insertItem(0, "XYZ", rotComboBoxItems::XYZ);
+	rotComboBox->insertItem(1, "X", rotComboBoxItems::X);
+	rotComboBox->insertItem(2, "Y", rotComboBoxItems::Y);
+	rotComboBox->insertItem(3, "Z", rotComboBoxItems::Z);
+	rotComboBox->insertItem(4, "None", rotComboBoxItems::NONE);
+	rotComboBox->setCurrentIndex(rotComboBoxItems::XYZ);
+	
 	connect(pauseButton,    &QAbstractButton::toggled, this, &ccGraphicalTransformationTool::pause);
 	connect(okButton,       &QAbstractButton::clicked, this, &ccGraphicalTransformationTool::apply);
 	connect(razButton,	    &QAbstractButton::clicked, this, &ccGraphicalTransformationTool::reset);
 	connect(cancelButton,   &QAbstractButton::clicked, this, &ccGraphicalTransformationTool::cancel);
+	connect(cancelButton,   &QAbstractButton::clicked, this, &ccGraphicalTransformationTool::cancel);
+	connect(cancelButton,   &QAbstractButton::clicked, this, &ccGraphicalTransformationTool::cancel);
```

Comment:
added twice?

Reply:
thanks, it was a copy paste to prepare the integration of forward/backward buttons... and it ends to be dead code